docs/taker.md

```diff
@@ -16,36 +15,39 @@ The taker CLI is an application that allows you to perform coinswaps as a taker.
 
 ### Start Bitcoin Core (Pre-requisite)
 
-`Taker` requires a **Bitcoin Core** RPC connection running on a **custom signet** for its operation(check demo doc](./demo.md)). To get started, you need to start `bitcoind`:
-
-> **Important:**  
-> All apps are designed to run on our **custom signet** for testing purposes. The DNS server that Taker connects to will also be on signet. While you can run these apps on other networks, there won't be any DNS available, so Taker won't be able to connect to the DNS server for getting maker's offers and can't do coinswap with makers. Alternatively, you can run your own DNS server on the network of your choice.
+`Taker` requires a **Bitcoin Core** RPC connection running on a **custom signet** for its operation(check [demo doc](./demo.md)). To get started, you need to start `bitcoind`:
 
 To start `bitcoind`:
 
 ```bash
 $ bitcoind
 ```
 
-**Note:** If you don’t have `bitcoind` installed or need help setting it up, refer to the [bitcoind demo documentation](./bitcoind.md).
-
+**Note:** If you don't have `bitcoind` installed or need help setting it up, refer to the [bitcoind demo documentation](./bitcoind.md).
 
 ### Usage
 
-Run the `taker` command to see the list of available commands and options.
+Run the `taker` command to see the list of available commands and options:
 
-```sh
+```bash
 $ ./taker --help
-coinswap 0.1.0
+```
+
+This will display a detailed guide about the app and its capabilities.
+
+#### **Output:**
+
+```bash
+coinswap 0.1.2
 Developers at Citadel-Tech
 A simple command line app to operate as coinswap client.
 
-The app works as regular Bitcoin wallet with added capability to perform coinswaps. The app requires
-a running Bitcoin Core node with RPC access.
-Use this faucet: http://xjw3jlepdy35ydwpjuptdbu3y74gyeagcjbuyq7xals2njjxrze6kxid.onion/(open in Tor browser) to get some signet coins.
+The app works as a regular Bitcoin wallet with the added capability to perform coinswaps. The app
+requires a running Bitcoin Core node with RPC access. It currently only runs on Testnet4. Suggested
+faucet for getting Testnet4 coins: https://mempool.space/testnet4/faucet
```

Comment:
Change this to custom signet faucet. Change the docs in the apps to fit the current behavior.